src/main/java/core/basesyntax/FileCreator.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.io.BufferedWriter;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.PrintWriter;
+
+public class FileCreator {
+
+    public static void createFile() {
+        String fileName = "/Users/Test_user/IdeaProjects/jv-fruit-shop/reportToRead.csv";
```

Comment:
Avoid using absolute paths to resources as it makes the code less portable. Use a relative path instead and consider passing it as a parameter to the method. - Bad: `String fileName = "/Users/Test_user/IdeaProjects/jv-fruit-shop/reportToRead.csv";` - Good: `public void createFile(String filePath) { ... }`

Reply:
not fixed, it should be in resources and save path to constant